src/esp/bindings/PhysicsObjectBindings.cpp

```diff
@@ -209,8 +209,9 @@ void declareRigidBaseWrapper(py::module& m,
 
       /* --- Geometry & Transformations --- */
 
-      .def_property_readonly("scale", &RigidBaseWrapper::getScale,
-                             ("Get the scale of the " + objType).c_str())
+      .def_property("scale", &RigidBaseWrapper::getScale,
+                    &RigidBaseWrapper::setScale,
```

Comment:
`setScale` is an expensive operation, right? To me, property getter/setters imply negligible runtime cost. You might consider an explicit python `set_scale` to discourage python users from doing lazy things like:
```
obs.scale = 1.0
if larger:
  obs.scale = 2.0
```